vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -227,10 +235,6 @@ def process_weights_after_loading(self, layer: Module) -> None:
             if self.quant_config.activation_scheme == "dynamic":
                 layer.input_scale = None
             elif self.quant_config.activation_scheme == "static":
-                if not all_close_1d(layer.input_scale):
-                    raise ValueError(
-                        "All the input_scales for the logical weights of a "
-                        f"layer must be equal. But got {layer.input_scale}")
```

Comment:
Why is this check not needed anymore?

Reply:
* We always initialize 3 scales in QKVColumnLinear
* In the case of fused QKV on disk, there is only 1 scale on disk for QKV
* In this PR, I load the scale from disk into the first index of the scales in QKVColumnLinear (and the other 2 are set to -inf)
* So it is no longer true that these all need to be "close"